### PR TITLE
Update create order endpoint

### PR DESCRIPTION
**Pull request by kiran-mastek**
Include secondary organizations in authorization check

Story: [AB#12959](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/12959)
Task: [AB#13899](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13899)

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Mime;
+using System.Security.Claims;
```
Comment (bjss-chris) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs:5]: Do we need `using System.Security.Claims;`?

**Review by bjss-chris — COMMENTED** [review]
Looks good – just one question.

**Comment by bjss-chris** [comment]
Also, now that you've moved `IsAuthorisedForOrganisation` to `ClaimsPrincipalExtensions` can we add a test for it to `ClaimsPrincipleExtensionsTests`?

File: src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs (hunk 1 above)
Comment (kiran-mastek) [reply src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs]: Good catch, no, removed

File: tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/ClaimsPrincipalExtensionsTests.cs [hunk 2]
```diff
@@ -73,5 +74,49 @@ internal static class ClaimsPrincipalExtensionsTests
 
             user.GetUserName().Should().Be(name);
         }
+
+        [Test]
+        [AutoData]
+        public static void IsAuthorisedForOrganisation_OrganisationIdIsPrimary_ReturnsTrue(            string name)
```
Comment (bjss-chris) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/ClaimsPrincipalExtensionsTests.cs:80]: Looks like we've got some superfluous white space 😃 

I'm surprised there wasn't a linting error.

**Review by bjss-chris — CHANGES_REQUESTED** [review]
Just some superfluous white space to remove – surprised weren't linting errors.

File: tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/ClaimsPrincipalExtensionsTests.cs (hunk 2 above)
Comment (kiran-mastek) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/ClaimsPrincipalExtensionsTests.cs:80]: corrected

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**